notebooks/ImvizExample.ipynb

```diff
@@ -497,8 +499,8 @@
    "source": [
     "# Add 100 randomly generated X,Y (0-indexed) w.r.t. reference image\n",
```

Comment:
```suggestion
    "# Add 50 randomly generated X,Y (0-indexed) w.r.t. reference image\n",
```

Reply:
Dropped it to 20 markers in the most recent commit.